src/js/view/floating-close-button.js

```diff
@@ -11,16 +10,24 @@ export default class FloatingCloseButton extends Events {
         this.element = createElement(floatingCloseButton(ariaLabel));
 
         this.element.appendChild(cloneIcon('close'));
-        this.ui = new UI(this.element).on('click tap enter', () => {
```

Comment:
Removing this will cause an error on line 149 of controls.js, as `floatCloseButton.on` will be undefined.

`        if (floatingConfig) {
            const floatCloseButton = new FloatingCloseButton(element, model.get('localization').close);
            const doNotForward = true;
            floatCloseButton.on(USER_ACTION, () => this.trigger('dismissFloating', { doNotForward }));

            if (floatingConfig.dismissible !== false) {
                addClass(this.playerContainer, 'jw-floating-dismissible');
            }
        }`